src/DynamoCore/Graph/Workspaces/WorkspaceModel.cs

```diff
@@ -2041,116 +2041,6 @@ internal bool containsInvalidInputSymbols()
             return this.Nodes.OfType<Nodes.CustomNodes.Symbol>().Any(node => !node.Parameter.NameIsValid);
         }
 
-        [Obsolete("Method will be deprecated in Dynamo 3.0.")]
-        private void SerializeElementResolver(XmlDocument xmlDoc)
-        {
-            Debug.Assert(xmlDoc != null);
-
-            var root = xmlDoc.DocumentElement;
-
-            var mapElement = xmlDoc.CreateElement("NamespaceResolutionMap");
-
-            foreach (var element in ElementResolver.ResolutionMap)
-            {
-                var resolverElement = xmlDoc.CreateElement("ClassMap");
-
-                resolverElement.SetAttribute("partialName", element.Key);
-                resolverElement.SetAttribute("resolvedName", element.Value.Key);
-                resolverElement.SetAttribute("assemblyName", element.Value.Value);
-
-                mapElement.AppendChild(resolverElement);
-            }
-            root.AppendChild(mapElement);
-        }
-
-        [Obsolete("Method will be deprecated in Dynamo 3.0.")]
-        protected virtual bool PopulateXmlDocument(XmlDocument xmlDoc)
```

Comment:
hmm... does this break anything to do with undo/redo?

Reply:
I need to check.

Reply:
There are no references to them. Looks like dead code to me.